gard_owl_ingest/mondo_mapping_status.py

```diff
@@ -96,61 +97,78 @@ def gard_mondo_mapping_status(
         'both')
     existing_df = existing_df.sort_values(['status', 'subject_id', 'in_gard', 'in_mondo'])
     existing_gard_mondo_mappings: List[CURIE] = existing_df[existing_df['status'] == 'both']['subject_id'].tolist()
     existing_df.to_csv(ANALYSIS_OUTDIR / f'gard_terms_mapping_status{file_suffix}.csv', index=False)
 
-    # gard.sssom.tsv
-    # - Add proxy mappings between GARD and Mondo, through Orphanet and OMIM
+    # gard-mondo.sssom.tsv: (i) GARD::Mondo mappings via proxy (Orphanet/OMIM), (ii) keep unmapped
+    # todo: should everything for gard-mondo.sssom.tsv be its own function?
+    # - sssom_proxyable_df: GARD::proxy mappings. Native from GARD.
     rows = []
-    # todo: include: prefix_map etc as comment
-    # todo: include: cols: subject_label, mapping_justification
-    for s, xref_dict in mappings.items():
+    for s, xref_dict in gard_native_mappings.items():
         for p, o_list in xref_dict.items():
             for o in o_list:
                 rows.append({
                     'subject_id': s,
                     'predicate_id': p,
                     'object_id': o,
                 })
-    sssom_proxy_df = pd.DataFrame(rows)
-    sssom_proxy_df2 = pd.merge(
-        sssom_proxy_df, mondo_sssom_df, how='left', left_on='object_id', right_on='mondo_object_id').fillna('')\
+    sssom_proxyable_df = pd.DataFrame(rows)
+
+    # - sssom_proxy_df: GARD::Mondo JOINed mappings via proxy (Orphanet/OMIM)
+    proxy_and_unmapped_df = pd.merge(
+        sssom_proxyable_df, mondo_sssom_df, how='left', left_on='object_id', right_on='mondo_object_id').fillna('')\
         .sort_values(by='subject_id')
-    del sssom_proxy_df2['mondo_object_id']
-    sssom_proxy_df2['mapping_justification'] = 'semapv:ManualMappingCuration'
-    proxied_gard_ids = set(sssom_proxy_df['subject_id'].tolist())
-    sssom_proxy_df2['gard_mondo_mapping_type'] = sssom_proxy_df2.apply(
+    del proxy_and_unmapped_df['mondo_object_id']
+    proxy_and_unmapped_df['mapping_justification'] = 'semapv:ManualMappingCuration'
+    # - proxy_existing: These were mappings that were made from Mondo to GARD before. We're likely going to ignore these
+    # completely. But every term on this list also is able to have a new proxy mapping between GARD and Mondo using our
+    # new algorithmic methods here.
+    # - unmapped: It's possible there may have been some previous Mondo::GARD mappings that were mapped that are
+    # actually in this list, but we are ignoring those now.
+    proxy_and_unmapped_df['gard_mondo_mapping_type'] = proxy_and_unmapped_df.apply(
         lambda x: 'proxy_existing' if x.subject_id in existing_gard_mondo_mappings
         else 'proxy_new' if x.mondo_id != ''
-        else 'proxy_new_via_other_path' if x.subject_id in proxied_gard_ids
-        else 'unexpected_unclassified', axis=1)
-    # - Add missing mappings
-    unmapped_ids: List[CURIE] = [y for y in in_gard if y not in proxied_gard_ids]
```

Comment:
Bugfix: I was reporting too few (incorrect list of) unmapped. `proxied_gard_ids` would have been better called `proxyable_gard_ids`. I confused myself. Anyway, I simplified the code and fixed.